src/notaryUtils.js

```diff
@@ -7,16 +7,24 @@ const getTransferState = require('./transferUtils').getTransferState
 const Payments = require('./payments')
 
 /**
+ * @returns {String} New case ID
+ */
+function createCaseID () {
+  return uuid()
```

Comment:
Any idea why we're using [`uuid4`](https://github.com/tracker1/node-uuid4) versus node-uuid, https://github.com/broofa/node-uuid? The latter seems to be more popular and tested.